services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouterController.java

```diff
@@ -268,6 +342,11 @@ public RouteTablesWebJson getVpcRouteTables(@PathVariable String projectid, @Pat
 
             routetables = this.routerService.getVpcRouteTables(projectid, vpcid);
 
+            if (routetables == null)
+            {
+                throw new RouterUnavailable();
+            }
+
```

Comment:
@DavidLiu506 Is this a `RouterUnavailable` or `RouteTableUnavailable`?

Reply:
Thanks!